packages/xstate-react/test/createActorContext.test.tsx

```diff
@@ -8,25 +8,6 @@ import {
 import { fireEvent, screen, render, waitFor } from '@testing-library/react';
 import { useSelector, createActorContext, shallowEqual } from '../src';
 
-const originalConsoleError = console.error;
-
-afterEach(() => {
-  console.error = originalConsoleError;
-});
-
-function checkConsoleErrorOutputForMissingProvider() {
```

Comment:
this was really just testing React's own behavior and that has changed, we don't need this